backend/src/main/java/gov/cdc/usds/simplereport/service/PatientRegistrationLinkService.java

```diff
@@ -0,0 +1,24 @@
+package gov.cdc.usds.simplereport.service;
+
+import gov.cdc.usds.simplereport.api.model.errors.InvalidPatientRegistrationLinkException;
+import gov.cdc.usds.simplereport.db.model.PatientRegistrationLink;
+import gov.cdc.usds.simplereport.db.repository.PatientRegistrationLinkRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+// NOTE: as of today, only those methods exposed to graphql endpoints have method-level security.
+// We will likely want to want security for the others in the near future.
+@Service("patientRegistrationLinkService")
+@Transactional(readOnly = false)
+public class PatientRegistrationLinkService {
+
+  @Autowired private PatientRegistrationLinkRepository prlrepo;
```

Comment:
Also a minor nitpick: broadly we prefer to use constructor injection instead of `@Autowired`, because it lends itself more easily to injecting mocks in unit tests. I'm guilty for writing many of the `@Autowired` fields in the early patient experience code, but Ben has since swayed me to using constructor injection exclusively 

See: https://reflectoring.io/constructor-injection/